docs/dev_guide/stability.rst

```diff
@@ -0,0 +1,100 @@
+Current status of sub-packages
+==============================
+
+SunPy has in variations in stability across sub-packages.
+This document summarizes the current status of the SunPy sub-packages, so that users understand where they might expect changes in future, and which sub-packages they can safely use for production code.
+
+The classification is as follows:
+
+{% raw %}
+.. raw:: html
+{% endraw %}
+   <style>
+         .planned:before {
+              color: #cbcbcb;
+              content: "⬤";
+         }
+         .dev:before {
+              color: #ffad00;
+              content: "⬤";
+         }
+         .stable:before {
+              color: #4e72c3;
+              content: "⬤";
+         }
+         .mature:before {
+              color: #03a913;
+              content: "⬤";
+         }
+         .pendingdep:before {
+              color: #a84b03;
+              content: "⬤";
+         }
+         .deprecated:before {
+              color: #ff0000;
+              content: "⬤";
+         }
+    </style>
+
+    <table align='center'>
+      <tr>
+        <td align='center'><span class="planned"></span></td>
```

Comment:
having just had a look at the rendered version of this, I would like a little more padding between the columns in this table, it looks pretty cramped on the built docs.

Reply:
this is *super* nitpicky, feel free to ignore it.

Reply:
Does it look cramped tho?

Reply:
![screenshot_20180321_145905](https://user-images.githubusercontent.com/1391051/37717420-6b1caefe-2d18-11e8-8c69-583afaa70cba.png)

I just mean the lack of space on the right side of these dots.

Reply:
OH THAT PART

Reply:
I said it was nit picky :laughing:

Reply:
TRY NOW